stock_management/__init__.py

```diff
@@ -1,31 +1,59 @@
 from abc import ABC, abstractmethod
 from enum import Enum
+import time
+import PySimpleGUI as sg
+
 
 class Status(Enum):
-    INITIALIZING = 1,
-    RUNNING = 2,
+    INITIALIZED = 1
+    RUNNING = 2
     FINISHED = 3
 
-class SessionType(Enum):
```

Comment:
For simplicity, I removed the TEST type of run. There is only 1 type of run for a given session.